### PR TITLE
[CTCP-4901] changes to add navigate links for House Consignment. Departure Transport Means

### PR DESCRIPTION
**Pull request by prashanthmamidi**

### CODE REVIEW

**Comment by prashanthmamidi** [comment]
image

**Comment by prashanthmamidi** [comment]
image

**Comment by prashanthmamidi** [comment]
image

File: app/navigation/houseConsignment/index/GrossWeightNavigator.scala [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+import com.google.inject.Singleton
+import models._
+import navigation.Navigator
+import pages._
+import pages.houseConsignment.index.GrossWeightPage
+import play.api.mvc.Call
+
+@Singleton
+class GrossWeightNavigator extends Navigator {
```
Comment (jackheslop96) [inline app/navigation/houseConsignment/index/GrossWeightNavigator.scala:27]: GrossWeightNavigator makes this sound a little niche. Really this is the `HouseConsignmentNavigator`, it just so happens that the `GrossWeightPage` is the only case

File: test/controllers/houseConsignment/index/GrossWeightControllerSpec.scala [hunk 2]
```diff
@@ -98,7 +98,7 @@ class GrossWeightControllerSpec extends SpecBase with AppWithDefaultMockFixtures
       val result = route(app, request).value
 
       status(result) mustEqual SEE_OTHER
-      redirectLocation(result).value mustEqual onwardRoute.url
+      redirectLocation(result).value mustEqual "/manage-transit-movements/unloading/AB123/transit-unloading-permission-discrepancies/house-consignment/1"
```
Comment (jackheslop96) [inline test/controllers/houseConsignment/index/GrossWeightControllerSpec.scala:101]: Needing to change these is a sign that you aren't adequately mocking the navigator. If you look in `AppWithDefaultMockFixtures` you'll see a bunch of fake navigators. You'll need to add ones for your new navigator classes

File: test/controllers/houseConsignment/index/departureMeansOfTransport/CountryControllerSpec.scala [hunk 3]
```diff
@@ -125,7 +125,7 @@ class CountryControllerSpec extends SpecBase with AppWithDefaultMockFixtures wit
       val result = route(app, request).value
 
       status(result) mustEqual SEE_OTHER
-      redirectLocation(result).value mustEqual onwardRoute.url
+      redirectLocation(result).value mustEqual "/manage-transit-movements/unloading/AB123/transit-unloading-permission-discrepancies/house-consignment/1"
```
Comment (jackheslop96) [inline test/controllers/houseConsignment/index/departureMeansOfTransport/CountryControllerSpec.scala:128]: Same as above

File: test/controllers/houseConsignment/index/departureMeansOfTransport/IdentificationControllerSpec.scala [hunk 4]
```diff
@@ -133,7 +133,7 @@ class IdentificationControllerSpec extends SpecBase with AppWithDefaultMockFixtu
 
       status(result) mustEqual SEE_OTHER
 
-      redirectLocation(result).value mustEqual onwardRoute.url
+      redirectLocation(result).value mustEqual "/manage-transit-movements/unloading/AB123/transit-unloading-permission-discrepancies/house-consignment/1"
```
Comment (jackheslop96) [inline test/controllers/houseConsignment/index/departureMeansOfTransport/IdentificationControllerSpec.scala:136]: Same as above

File: test/controllers/houseConsignment/index/departureMeansOfTransport/IdentificationNumberControllerSpec.scala [hunk 5]
```diff
@@ -113,7 +113,7 @@ class IdentificationNumberControllerSpec extends SpecBase with AppWithDefaultMoc
       val result = route(app, request).value
 
       status(result) mustEqual SEE_OTHER
-      redirectLocation(result).value mustEqual onwardRoute.url
+      redirectLocation(result).value mustEqual "/manage-transit-movements/unloading/AB123/transit-unloading-permission-discrepancies/house-consignment/1"
```
Comment (jackheslop96) [inline test/controllers/houseConsignment/index/departureMeansOfTransport/IdentificationNumberControllerSpec.scala:116]: Same as above

**Review by jackheslop96 — APPROVED** [review]

**Pull request merged**